wireguard/config.py

```diff
@@ -67,6 +69,10 @@ def dns(self):
         Returns the DNS settings of the given peer for the config file
         """
 
+        # do not write empty DNS = entry
+        if not bool(self._peer.dns):
+            return None
+
```

Comment:
This is a good catch. We should add this to other attributes as well to prevent them from outputting at all when empty. However, for most attributes, we should be checking `if self._peer.dns is None` as `None` is the unset value.

I don't expect this to be added to the other attributes as part of this PR. We can do it after, as well as add some more tests to verify the functionality.

Reply:
In my case it returned a "DNS = " entry that was the reason for thinking about it